test/extensions/filters/network/redis_proxy/redis_proxy_integration_test.cc

```diff
@@ -462,13 +462,14 @@ void RedisProxyIntegrationTest::initialize() {
   setUpstreamCount(num_upstreams_);
   setDeterministic();
   config_helper_.renameListener("redis_proxy");
+  on_server_ready_function_ = [this](Envoy::IntegrationTestServer& test_server) {
+    mock_rng_ = dynamic_cast<Runtime::MockRandomGenerator*>(&(test_server.server().random()));
+    // Abort now if we cannot downcast the server's random number generator pointer.
+    ASSERT_TRUE(mock_rng_ != nullptr);
+    // Ensure that fake_upstreams_[0] is the load balancer's host of choice by default.
+    ON_CALL(*mock_rng_, random()).WillByDefault(Return(0));
```

Comment:
This is the default, so none of this is necessary for these cases. Can you remove this? (I just noticed this because I need this for a test I'm working on).

Reply:
ok, removed.

Reply:
Sorry I mean actually remove all of this code that sets up to return 0 from existing tests. It's the default behavior.

/wait